src/styles/sidebar/components/PaginationNavigation.scss

```diff
@@ -0,0 +1,50 @@
+@use '../../mixins/buttons';
+@use "../../mixins/layout";
+@use "../../mixins/utils";
+@use "../../variables" as var;
+
+.PaginationNavigation {
+  @include utils.font--large;
+  @include layout.row;
+
+  &__relative {
+    // Set a minimum width for the previous and next button containers so that
+    // there is no jumping around as they appear and disappear.
+    min-width: 5em;
```

Comment:
I assume the "5" is 4 chars for "Prev" and "Next" plus one for the arrow? Is that enough?

Reply:
Refactored/gone.